torba/client/basedatabase.py

```diff
@@ -321,12 +326,8 @@ def _transaction(conn: sqlite3.Connection, tx: BaseTransaction, address, txhash,
         return self.db.run(_transaction, tx, address, txhash, history)
 
     async def reserve_outputs(self, txos, is_reserved=True):
-        txoids = [txo.id for txo in txos]
-        await self.db.execute(
-            "UPDATE txo SET is_reserved = ? WHERE txoid IN ({})".format(
-                ', '.join(['?']*len(txoids))
-            ), [is_reserved]+txoids
-        )
+        txoids = ((is_reserved, txo.id) for txo in txos)
```

Comment:
Is there a specific reason to use a tuple generator instead of list comprehension here?